sdk/firstload/rollup.config.js

```diff
@@ -22,7 +22,19 @@ const basePlugins = [
 	consts({
 		publicGraphURI: process.env.PUBLIC_GRAPH_URI,
 	}),
-	resolve({ browser: true }),
+	resolve({
+		browser: true,
+		// @highlight-run/client is a private package not published to npm, so
+		// listing it in package.json would break end users.
+		// Instead, we add root node_modules as a resolution path so it gets
+		// resolved as an internal module and included directly in the bundle.
+		// An alternative to the previous solution using relative paths that
+		// point outside package root described here:
+		// https://www.highlight.io/blog/publishing-private-pnpm-monorepo
+		modulePaths: ['../../node_modules'],
+		// Need to override this to add .ts and .tsx as valid resolution exts.
+		extensions: ['.mjs', '.js', '.json', '.node', '.ts', '.tsx'],
+	}),
```

Comment:
wow, this is very clever @lewisl9029 ! I had thought the relative imports were the only way to accomplish this before. i've always been concerned that doing some tweaking of the rollup settings would just make firstload inling @highlight-run/client and blow up the bundle, but that doesn't look like it is the case
<img width="890" alt="Screenshot 2023-04-07 at 10 19 46 AM" src="https://user-images.githubusercontent.com/1351531/230650443-a0f67034-9fe3-4385-a1e5-4e2600025b9c.png">

Reply:
Wouldn't have known about this if I didn't spend the last few months banging my head against bundler configs for the huge long tail of problematic packages haha... 😅